### PR TITLE
fix(angular): enable event names with forward slash

### PR DESCRIPTION
**Pull request by alexprudhomme**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally for affected output targets
- [x] Tests (`npm test`) were run locally and passed
- [x] Prettier (`npm run prettier`) was run locally and passed

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue URL:

## What is the new behavior?

- enables event names with forward slash
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/angular-output-target/src/generate-angular-component.ts [hunk 1]
```diff
@@ -188,7 +188,7 @@ export const createComponentTypeDefinition = (
   const eventTypes = publicEvents.map((event) => {
     const comment = createDocComment(event.docs);
     let eventName = event.name;
-    if (event.name.includes('-')) {
+    if (/[-/]/.test(event.name)) {
       // If an event name includes a dash, we need to wrap it in quotes.
```
Comment (tanner-reits) [inline packages/angular-output-target/src/generate-angular-component.ts:192]:
```suggestion
      // If an event name includes a dash or a forward slash, we need to wrap it in quotes.
```

**Review by tanner-reits — CHANGES_REQUESTED** [review]
One little comment update, then should be good!

**Review by tanner-reits — APPROVED** [review]

**Comment by tanner-reits** [comment]
@alexprudhomme Can you actually add a test for this to [this file](https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/angular-output-target/__tests__/generate-angular-component.spec.ts)?

**Comment by alexprudhomme** [comment]
done. thanks for merging these !

**Review by brandyscarney — APPROVED** [review]

**Pull request merged**